src/assets/scripts/client/tutorial/TutorialView.js

```diff
@@ -403,6 +403,40 @@ export default class TutorialView {
         });
 
         this.tutorial_step({
+            title: 'Say Route',
+            text: ['With the capability to edit the route, you obviously will need a way to know what their current route is. Typically, this is displayed',
+                   'in the flight progress strip. However, to preserve screen space, you will instead have to check the route by this command &lsquo;route&rsquo;',
+                   'and the route will print out above the command bar.'
+               ].join(' '),
+            parse: (t) => {
+                if (prop.aircraft.list.length <= 0) {
+                    return t;
+                }
+
+                return t.replace('{CALLSIGN}', prop.aircraft.list[0].callsign);
+            },
+            side: 'left',
+            position: tutorial_position
+        });
+
+        this.tutorial_step({
+            title: 'Proceed Direct',
+            text: ['The command &lsquo;pd&rsquo; instructs the aircraft to go direct to a navigational fix, taking a shortcut. For example, if an',
```

Comment:
instead of:
>The command &lsquo;pd&rsquo; instructs the aircraft to go direct to a navigational fix, taking a shortcut

we should use:
>The proceed direct command &lsquo;pd&rsquo; instructs an aircraft to go directly to a waypoint in the flight plan.